src/instructlab/model/evaluate.py

```diff
@@ -60,6 +60,16 @@ def get_evaluator(
     otherwise raises an exception for the missing values
     """
 
+    # ensure user is passing full safetensors if they specify a local directory
+    # TODO: also allow GGUF once the following is resolved: https://github.com/instructlab/eval/issues/50
+    if model is not None and os.path.isdir(model):
+        if not backends.is_model_safetensors(pathlib.Path(model)):
+            click.secho(
+                "Evaluation can currently only be used with safetensors",
```

Comment:
This is only true for MMLU right?

Reply:
Yes I think so - the check may make more sense within the MMLU benchmark param check section of the code - some discussion around this in the related bug as well: https://github.com/instructlab/instructlab/issues/1792

Reply:
+1 Having it inside the param checking block applying to mmlu and mmlu_branch above is probably best.  Until we can move serving out of lm_eval.

Reply:
+1 to this, it seems like the only code changes should be made around line ~115 in main where the code paths that run MMLU and MMLU-Branch are.

Reply:
and the log statement should reflect that MMLU and MMLU-Branch can currently only be run with safetensors.

Reply:
I will refactor to do this!